src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -44,7 +44,164 @@ paths:
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    get:
+      summary: Returns a single certificate
+      description: Returns a single certificate
+      tags:
+        - Single Certificate
+      responses:
+        '200':
+          $ref: '#/components/responses/Certificate'
+        '404':
+          description: Not Found
+      operationId: get-certificate-by-application-id
+      security:
+        - eroUserCognitoUserPoolAuthorizer: [ ]
+      x-amazon-apigateway-integration:
+        type: HTTP_PROXY
+        uri: '${base_uri}/eros/{eroId}/applications/{applicationId}'
+        requestParameters:
+          integration.request.path.eroId: method.request.path.eroId
+          integration.request.path.applicationId: method.request.path.applicationId
+        responseParameters:
+          method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+          method.response.header.Access-Control-Allow-Methods: '''*'''
+          method.response.header.Access-Control-Allow-Origin: '''*'''
+        connectionType: VPC_LINK
+        connectionId: '${vpc_connection_id}'
+        httpMethod: GET
+
 components:
+  #
+  # Schema and Enum Definitions
+  # --------------------------------------------------------------------------------
+  schemas:
+    CertificateResponse:
+      title: CertificateResponse
+      type: object
+      description: A response containing a single Certificate
+      properties:
+        vacNumber:
+          type: string
+          pattern: '^[0-9AC-HJ-NP-RT-Z]{20}$'
+          description: The unique identifier of the Certificate. A 20 character alphanumeric string.
+          example: DEV1W0XDH1368LWE1J40
+        printRequests:
+          type: array
+          description: List of print requests for the certificate.
+          items:
+            $ref: '#/components/schemas/PrintRequest'
+      required:
+        - vacNumber
+        - printRequests
+    PrintRequest:
```

Comment:
As above, is this a full Print Request object, or some kind of summary?
If we choose to rename the object here, then perhaps we also need to rename the property in `Certificate` (or whatever its new name is 😁 ) in a similar way ?

Also, I wonder if the description of the fields in this object need to be in the past tense? `UserId of the user who made the print request`, `The timestamp in ISO-8601 UTC that the print request status update was received` etc
Otherwise someone might look at the schemas in this spec, find this, and assume we can create a new PrintRequest somehow with some API endpoint that uses this object?

What do you think?

Reply:
Thanks, for the suggestions, @NathanRussellValtech ,
I agree for the summary renaming and we can make the descriptions past sense to avoid any confusions.
I pushed the changes accordingly.